src/commandwindow/ExecutionCommandProvider.ts

```diff
@@ -297,4 +297,29 @@ export default class ExecutionCommandProvider {
 
         void this._mvm.feval('cd', 0, [uri.fsPath]);
     }
+
+    /**
+     * Implements the open file action
+     * @param uri The file path to the file that should be opened
+     * @returns
+     */
+    async handleOpenFile(uri: vscode.Uri): Promise<void> {
```

Comment:
Looks like a space is missing before the `(`, which is causing the linting to fail

Reply:
Should be taken care of!

Reply:
Thanks!